rero_ils/modules/documents/serializers/base.py

```diff
@@ -64,22 +66,30 @@ def _get_pid(self) -> str:
         """Return reference id."""
         return self.record['pid']
 
+    def _is_masked(self) -> str:
+        """Return masked information."""
+        return 'Yes' if self.record.get('_masked') else 'No'
+
     def _get_title(self) -> str:
         """Return first title."""
         return next(
             filter(lambda x: x.get('type') == 'bf:Title',
                    self.record.get('title')), {}
         ).get('_text')
 
+    def _get_series_statement(self) -> list[str]:
+        """Return series statement title."""
+        return [
+            data['value']
+            for Statement in self.record.get('seriesStatement', [])
```

Comment:
is the upper "S" is necessary ? seems typo no ?

Reply:
Uuups...